pubsub/awspubsub/awspubsub.go

```diff
@@ -402,6 +404,11 @@ func openSubscription(ctx context.Context, client *sqs.SQS, qURL string) driver.
 	return &subscription{client: client, qURL: qURL}
 }
 
+const (
+	// How often ReceiveBatch should poll if no messages are available.
```

Comment:
Comment and name are not quite accurate. It's more like how long ReceiveBatch should wait before returning no messages. The polling is done in pubsub.Receive.

Reply:
Better?